pages/linux/swaylock.md

```diff
@@ -0,0 +1,36 @@
+# swaylock
+
+> Swaylock is a screen locking utility for Wayland compositors.
```

Comment:
```suggestion
> Screen locking utility for Wayland compositors.
```

Reply:
Sure, happy to make some changes brother.